File: C++L5/Digits.cpp

#include "Digits.h"

size_t Digits::getLength() const {
    return value.length();
}

std::string Digits::removeChar() {
    std::string result;
    for (char ch : value) {
        if (ch != '5') result += ch;
    }
    return result;
}

File: C#L5/Digits.cs
namespace StringLibrary
{
    public class Digits : StringBase
    {
        public Digits(string val) : base(val) { }

        public override int GetLength() => value.Length;

        public override string RemoveChar()
        {
            return value.Replace("5", "");
        }
    }
}
